doc/Docker.md

```diff
@@ -0,0 +1,51 @@
+Docker
+=====
+
+* [Home](help)
+
+
+Auto-Install
+---
+
+With the autoinstall function of friendice, you can automatically create docker environments, where some of the important configs (f.e the db credentials) are saved inside your volume beside your friendica data.  
+
+### Basics
+
+Docker automatically creates writable [containers](https://docs.docker.com/get-started/part2/) for friendica, which means that the data doesn't persist when that container no longer exists.
+Therefor, you have to manage your data with the help of [mountpoints](https://docs.docker.com/storage/).
+
+The goal of the auto-install feature is that the configuration of your data is stored inside your mountpoint, so if you delete the container (or lost your docker environment because of failures) you don't have to worry about your configuration. It is saved with your data.   
```

Comment:
- I don't understand the sentence, is the auto-install feature creating and managing the mountpoint?
- `lose` instead of `lost`
- `Docker`

Reply:
No, the auto-install feature uses the mountpoint for storing the configuration. Docker itself is creating and managing the mountpoint.

I tried to explain that the configuration file is important so you better store it inside a mountpoint. And the mountpoints are a mandatory part of backups (because they hold the data).

Do you have a better description for it? :-/

Reply:
`The goal of the auto-install feature is to use the configuration file stored inside your mountpoint to install Friendica, so if you delete the container (or lose your Docker environment because of failures) you won't have to start the install from scratch.`

?

Reply:
Thanks.. Much better to read!